runtime/src/main/java/org/corfudb/runtime/clients/ManagementClient.java

```diff
@@ -130,4 +132,20 @@ public QueryResponse queryRequest(UUID workflowId) throws Exception {
                 .payloadMsg(req));
         return (QueryResponse) resp.get().getResponse();
     }
+
+    public CreateWorkflowResponse removeNode(String endpoint) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)